src/main/webapp/app/shared/detail-overview-navigation-bar/detail-overview-navigation-bar.component.ts

```diff
@@ -10,15 +11,12 @@ export class DetailOverviewNavigationBarComponent implements AfterViewInit {
     sectionHeadlines: { id: string; translationKey: string }[];
 
     @HostListener('window:resize')
-    onResize() {
-        setTimeout(() => {
-            const headerHeight = (document.querySelector('jhi-navbar') as HTMLElement).offsetHeight;
-            document.documentElement.style.setProperty('--header-height', `${headerHeight - 2}px`);
-        });
+    onResizeAddDistanceFromStatusBarToNavbar() {
+        updateHeaderHeightScssVariableBasedOnNavbar();
```

Comment:
Personal opinion, but I think this function name is quite long. I'd rather go with something like `updateHeaderHeight`